dialog/entry.go

```diff
@@ -0,0 +1,111 @@
+package dialog
+
+import (
+	"fyne.io/fyne"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+)
+
+// EntryDialog is a variation of a dialog which prompts the user to enter some
+// text.
+type EntryDialog struct {
+	*dialog
+
+	entry *widget.Entry
+
+	confirmButton *widget.Button
+
+	onClosed func()
+}
+
+// SetText changes the current text value of the entry dialog, this can
+// be useful for setting a default value.
+func (i *EntryDialog) SetText(s string) {
+	i.entry.SetText(s)
+}
+
+// GetText returns the current text value entered in the entry dialog.
```

Comment:
I lot has changed in this area so I wanted to check before this wraps up - you're sure this API is useful?

Reply:
Agreed, I don't think there is much use case, so I went ahead and nuked it in 775dc541.